### ISSUE TITLE
QE 1.0.1: When you delete a QE with a Note, get an error from Foundry and the Note isn't deleted

### ISSUE THREAD

**Issue by spetzel2020** [spetzel2020/quick-encounters#84]
Not the same as Issue #57 
```
EncounterNote.js:130 Uncaught (in promise) TypeError: scene.deleteEmbeddedEntity is not a function
    at delete (EncounterNote.js:130:19)
    at Function._call (foundry.js:294:14)
    at Function.callAll (foundry.js:253:12)
    at ClientDatabaseBackend.callback (foundry.js:10490:13)
    at foundry.js:10429:43
    at Array.map (<anonymous>)
    at ClientDatabaseBackend._handleDeleteDocuments (foundry.js:10429:33)
    at ClientDatabaseBackend._deleteDocuments (foundry.js:10335:17)
    at async Function.deleteDocuments (document.mjs:410:21)
    at async JournalEntry.delete (document.mjs:471:21)
```

**Comment by spetzel2020** [comment]
Fixed in 1.0.1

**Comment by spetzel2020** [comment]
Need to verify that this was fixed in 1.0.1.
